### PR TITLE
SSB: QC changes and Description updates

### PR DESCRIPTION
**Pull request by ImLycan**
Everything is tested and works, all new short descriptions fit. Just need a qc ~~im only really doing this because i have nothing else to do~~

### CODE REVIEW

File: mods/ssb/moves.js [hunk 1]
```diff
@@ -3464,8 +3468,8 @@ let BattleMovedex = {
 	rotate: {
 		accuracy: 100,
 		category: "Status",
-		desc: "The next Pokemon will switch out after using its move if the move is successful.",
-		shortDesc: "Next Pokemon will switch after using its move.",
+		desc: "The user's replacement will switch out after using their move on the next turn if the replacement’s move is successful",
```
Comment (KrisXV) [inline mods/ssb/moves.js]:
```suggestion
		desc: "The user's replacement will switch out after using their move on the next turn if the replacement's move is successful.",
```
don't use curly apostrophes also there should be a period at the end of the sentence

File: mods/ssb/moves.js [hunk 2]
```diff
@@ -3666,7 +3670,7 @@ let BattleMovedex = {
 		accuracy: true,
 		category: "Status",
 		desc: "Moves all hazards that are on the user's side of the field to the foe's side of the field. Sets Stealth Rock on the foe's side, after which the user switches out.",
-		shortDesc: "Moves hazards to foe's side. Switches out.",
+		shortDesc: "Hazards to foe's side. Stealth Rock. Switches out.",
```
Comment (KrisXV) [inline mods/ssb/moves.js]:
```suggestion
		shortDesc: "Hazards -> foe side. Set SR. User switches out.",
```

File: mods/ssb/random-teams.js [hunk 3]
```diff
@@ -488,7 +488,7 @@ class RandomStaffBrosTeams extends RandomTeams {
 			},
 			'Rory Mercury': {
 				species: 'Charjabug', ability: 'Recharge', item: 'Eviolite', gender: 'M',
-				moves: ['Spikes', ['Leech Life', 'U-turn'], ['Poison Fang', 'Nuzzle']],
+				moves: ['Spikes', 'Leech Life', ['Poison Fang', 'Nuzzle']],
```
Comment (KrisXV) [inline mods/ssb/random-teams.js]: I don't know why you're removing U-turn, it was never said to be removed

Comment (ImLycan) [reply mods/ssb/random-teams.js]: Talked on discord

**Pull request merged**